docs/products/clickhouse/concepts/clickhouse-tiered-storage.rst

```diff
@@ -0,0 +1,83 @@
+Tiered storage in Aiven for ClickHouse®
+=======================================
+
+.. important::
+
+    Aiven for ClickHouse® tiered storage is a :doc:`limited availability feature </docs/platform/concepts/beta_services>`. If you're interested in trying out this feature, contact the sales team at `sales@Aiven.io <mailto:sales@Aiven.io>`_.
+
+Discover the tiered storage capability in Aiven for ClickHouse®. Learn how it works and explore its use cases. Check why you might need it and what benefits you get using it.
+
+Overview
+--------
+
+The tiered storage feature introduces a method of organizing and storing data in two tiers for improved efficiency and cost optimization. The data is automatically moved to an appropriate tier based on your database's local disk usage. On top of this default data allocation mechanism, you can control the tier your data is stored in using custom data retention periods.
+
+The tiered storage in Aiven for ClickHouse consists of the following two layers:
+
+SSD - the first tier
+  Fast storage device with limited capacity, better suited for fresh and frequently queried data, relatively costly to use
+
+Object storage - the second tier
+  Affordable storage device with unlimited capability, better suited for historical and more rarely queried data, relatively slower
+
+Why use it
+----------
+
+By :doc:`enabling </docs/products/clickhouse/howto/enable-tiered-storage>` and properly :doc:`configuring </docs/products/clickhouse/howto/configure-tiered-storage>` the tiered storage in Aiven for ClickHouse, you can use storage resources efficiently and, therefore, significantly reduce costs of storing data ingested into an Aiven for ClickHouse instance.
```

Comment:
Given the change driven by design feedback, tiered storage will be enabled for all. So can shorten this sentence.